dipy/tracking/fbcmeasures.pyx

```diff
@@ -0,0 +1,334 @@
+                for pointId in range(streamlines_length[lineId] - 1):
+                    score_mp[lineId] = 0.0
+                    for lineId2 in range(numberOfFibers):
+                    
+                        # skip lfbc computation with itself
+                        if lineId == lineId2:
+                            continue
+                            
+                        for pointId2 in range(streamlines_length[lineId2] - 1):
+                            # compute displacement
+                            xd_mp[lineId] = int(streamlines[lineId, pointId, 0] 
+                            - streamlines[lineId2, pointId2, 0] + 0.5)
+                            yd_mp[lineId] = int(streamlines[lineId, pointId, 1] 
+                            - streamlines[lineId2, pointId2, 1] + 0.5)
+                            zd_mp[lineId] = int(streamlines[lineId, pointId, 2] 
+                            - streamlines[lineId2, pointId2, 2] + 0.5)
+                            
+                            # if position is outside the kernel bounds, skip
+                            if xd_mp[lineId] > hn or -xd_mp[lineId] > hn or \
+                               yd_mp[lineId] > hn or -yd_mp[lineId] > hn or \
+                               zd_mp[lineId] > hn or -zd_mp[lineId] > hn:
+                                continue
+                            
+                            # grab kernel value from LUT
+                            score_mp[lineId] += \
+                                        lut[streamlines_nearestp[lineId, pointId], 
+                                        streamlines_nearestp[lineId2, pointId2], 
+                                        hn+xd_mp[lineId], 
+                                        hn+yd_mp[lineId], 
+                                        hn+zd_mp[lineId]]  # ang_v, ang_r, x, y, z
+                                    
+                    streamline_scores[lineId, pointId] = score_mp[lineId]
+
+        if have_openmp and num_threads is not None:
+            openmp.omp_set_num_threads(all_cores)
+        
+        # Save LFBC as class member
+        self.streamlines_lfbc = streamline_scores
+        
+        # compute RFBC for each fiber
+        self.streamlines_rfbc = compute_rfbc(streamlines_length, 
+                                              streamline_scores)
+        
+def compute_rfbc(streamlines_length, streamline_scores):
+    """ Compute the relative fiber to bundle coherence (RFBC)
+
+    Parameters
+    ----------
+    streamlines_length : 1D int array
+        Contains the length of each streamline
+    streamlines_scores : 2D double array
+        Contains the local fiber to bundle coherence (LFBC) for each streamline 
+        element.
+
+    Returns
+    ----------
+    output: normalized lowest average LFBC region along the fiber
+    """
+
+    # finds the region of the fiber with minimal length if 7 points in which the
+    # LFBC is the lowest
```

Comment:
The number 7 was something we found experimentally a long time ago, unfortunately I don't have good evidency why. However it's now available as a parameter to the user.